Makefile.dune

```diff
@@ -26,6 +26,12 @@ states: voboot
 world: voboot
 	dune build $(JBOPT) @install
 
+apidoc:
+        # Ugly workaround for https://github.com/ocaml/odoc/issues/148
+	mv checker/dune checker/dune.disabled
+	dune build $(JBOPT) @doc
+	mv checker/dune.disabled checker/dune
```

Comment:
How well does this work when dune fails?

Reply:
Yeah this should go away indeed.